### PR TITLE
Add base layout

### PR DESCRIPTION
**Pull request by davelange**
Why:
 - New layout has a sidebar nav wrapping all pages except Onboarding

How
 - Add a BaseLayout and Sidebar components that wrap the pages content. Not styled yet.
 - Move the wallet widget and behaviour in to the BaseLayout, as it will be present on all these pages.

### CODE REVIEW

**Comment by davelange** [comment]
@juliangruber while the other PRs are pending, I prepared this one to setup the new app layout structure.

File: renderer/src/App.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import Sentry from 'src/components/Sentry'
 import Plausible from 'src/components/Plausible'
 import { HelmetProvider, Helmet } from 'react-helmet-async'
 import { ROUTES } from 'src/lib/routes'
+import BaseLayout from 'src/components/BaseLayout'
```
Comment (juliangruber) [inline renderer/src/App.tsx]: can we call this `Layout` instead?

**Review by juliangruber — CHANGES_REQUESTED** [review]
Nice! I've pinged Miro on Slack about these reviews

**Comment by juliangruber** [comment]
could you please recreate this PR on the main repo?

**Pull request closed without merging**